aces/Loris
# Add month input selector to datepicker polyfill

## Brief summary of changes

Right now, on Firefox, a custom loris date picker will be rendered for every date element regardless of whether dateFormat = YM or YMD. This means that if dateFormat='YMD', every date element will render the firefox date picker AND the loris date picker. The changes made in this PR makes it so that a loris date picker is only rendered on the date element for which firefox has no datepicker if dateFormat = 'YM'.

#### Testing instructions (if applicable)

1. On minor, test date element in Firefox when Date Format in Configurations module set to 'YMD'. You will see that there are two date pickers. Change date format to 'YM', and you will only see one date picker (the loris one because Firefox doesn't support input type month).
2. On this branch, test date element again in Firefox when date format = 'YMD'. you should only see the firefox date picker. When date format = 'YM', same date picker renders as before.

#### Links to related tickets (GitHub, Redmine, ...)

*
